pymove/preprocessing/segmentation.py

```diff
@@ -276,14 +316,15 @@ def by_speed(
 
 
 def by_time(
-        move_data,
-        label_id=TRAJ_ID,
-        max_time_between_adj_points=900.0,
-        drop_single_points=True,
-        label_new_tid="tid_time",
-        inplace=True
+    move_data,
+    label_id=TRAJ_ID,
+    max_time_between_adj_points=900.0,
+    drop_single_points=True,
+    label_new_tid="tid_time",
+    inplace=True,
 ):
-    """Segments the trajectories into segments based on time.
+    """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Section underline should match the length of its name (Expected 7 dashes in section 'Returns', got 6) (D409)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5421338)